src/share/share.interface.ts

```diff
@@ -0,0 +1,11 @@
+import { IFile, IUser, Permission } from '../helpers/tmp.types';
+
+export default interface IShare{
+  id?: any;
+  file: IFile | String;
+  permissions: Permission;
+  from: IUser | String;
+  to: IUser | String;
```

Comment:
So, only one recipient? we will have as many shares as there are people who were shared to?